enzyme/Enzyme/ActivityAnalysis.cpp

```diff
@@ -1899,7 +1950,7 @@ bool ActivityAnalyzer::isInstructionInactiveFromOrigin(TypeResults &TR,
       if (!ci->onlyAccessesArgMemory() && !ci->doesNotAccessMemory()) {
 
         Function *called = getFunctionFromCall(ci);
-        if (!called || (!isCertainPrintMallocOrFree(called) &&
+        if (!called || (!isCertainCacheable(called) &&
```

Comment:
This code says that a call to a memory free libm function, allocation, free, or print, doesn't change the activity of a global.

This code doesn't have anything to do with cacheability.

Reply:
Undone